wca-regulations.md

```diff
@@ -291,6 +291,10 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 11a3) Equipment malfunction.
 - 11b) If an incident occurs, the WCA Delegate determines an impartial and appropriate course of action.
 - 11d) If the WCA Regulations are not fully clear or if the incident is not covered by the WCA Regulations, then the WCA Delegate must make a decision based on fair sportsmanship (also see [Regulation 11e3](regulations:regulation:11e3)).
+  - 11d1) The WCA Delegate may waive a penalty or award an extra attempt even when Regulation does not directly specify the use of their discretion. This must be noted in the Delegate report, and satisfy the following conditions:
+    - 11d1a) The competitor did not cause the incident, or did not cause it intentionally.
+    - 11d1b) The incident did not give the competitor a significant advantage for their result.
```

Comment:
```suggestion
    - 11d1b) The incident did not give the competitor an unfair advantage for their result.
```

Reply:
Good idea, that is more consistent with other with other Regulations.